### ISSUE TITLE
dokku

### ISSUE THREAD

**Issue by edwardvalentini** [arunoda/meteor-up-legacy#622]
has anyone tried to use mupx with dokku https://github.com/progrium/dokku  - to get a heroku style deployment strategy?  I would be interested in any experience people have had with this

**Comment by zimt28** [comment]
Nope, and I don't think that's gonna work. But I'm using [meteord](https://github.com/meteorhacks/meteord) with dokku :)
All you have to do is add a `Dockerfile` like

```
FROM meteorhacks/meteord:onbuild
EXPOSE 80
```

to your app and then `git push` to your dokku server.

**Comment by zimt28** [comment]
One more thing: When you create a dokku server (which you can easily do using DigitalOcean, they've got an image just to get started), don't use the smallest server. Building the docker images often failed when I pushed, using the 10$ version works great.

**Comment by edwardvalentini** [comment]
cool - thank you!
